types/httptoolkit__esm/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Project: https://github.com/httptoolkit/esm
```

Comment:
Please remove this header

Reply:
Hi @jakebailey,

I've removed the `// Project ...` line - did you also want the line after that removed?

Reply:
Yes, all of it.

Reply:
Done :)